ui/src/main/webapp/src/app/analysis-context/analysis-context-form.component.html

```diff
@@ -89,8 +89,10 @@ <h2 *ngIf="!project" i18n>Loading...</h2>
                     <ng-container i18n>Identifying packages...</ng-container>
                 </p>
             </div>
-            <wu-js-tree-wrapper [treeNodes]="packageTree" [(selectedNodes)]="analysisContext.includePackages">
-            </wu-js-tree-wrapper>
+            <div class="fixed-height">
+                <wu-js-tree-wrapper [treeNodes]="packageTree" [(selectedNodes)]="analysisContext.includePackages">
+                </wu-js-tree-wrapper>
+            </div>
```

Comment:
I tried it but it doesn't work and the tree kept getting more space in the page.